_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -311,6 +362,36 @@
 	},
 /turf/open/floor/plasteel,
 /area/security/prison)
+"aaY" = (
+/obj/effect/turf_decal/bot,
+/obj/structure/closet/crate/secure/weapon{
```

Comment:
This is kinda a big buff but meta station has it to so meh

Reply:
wait shit do these things have armor i was under the impression they were just flavor

might varedit them then

Reply:
the formal clothes? I gave them armor per request from someone lol